node/silkworm/common/log.hpp

```diff
@@ -104,4 +104,17 @@ using Message = LogBuffer<Level::kNone>;
 
 }  // namespace silkworm::log
 
+#define LOG(level_)                               \
```

Comment:
LOG is too similar to the keyword for a logarithmic function. (Which I now realize is also for the `log` namespace)
Maybe change to SILK_LOGGING_LEVEL ?

Reply:
I agree that at least the name of this macro should start with `SILK_` or `SILKWORM_`. More rationale at https://google.github.io/styleguide/cppguide.html#Preprocessor_Macros

Reply:
You're right! Maybe `SILK_LOG_LINE` fits better? Just because it's not a logging level, it's a log line (it receives the `level_` as parameter)

Reply:
Well actually is not a log line either: it eventually prepares the buffer to receive logging data.
My vote is for `SILK_LOGBUFFER(silkworm::log::Level::xxx)`  which reminds sort of a template.
But your call on this